server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -119,22 +119,22 @@ function uploadDocument($token, $firstName, $lastName, $emailAddress, $phoneNumb
 		}
 
 		// Check if file is Fillable Form 14446 and, if so, that it has changed
-		if (preg_match('/(.*)2020_F14446(.*)\.pdf(.*)/i', $uploadedFileName)) {
```

Comment:
Haha I see what you mean about this... This is a hard one, since we need to update these documents yearly too (probably?). The most obvious thing to me would be adding a test that verifies `$currentYear_F14446` passes this regex (just as a reminder for people to update it by failing a test), but we don't have any testing framework in this repo 🙈 

I don't think it's crazy to include the date in this regex to prevent an older version from being uploaded 🤷 That seems a lot simpler / easier than having to have some back-and-forth with the client/tax preparer when the preparer notices the file is the wrong one...

We should probably just put together a "yearly checklist" of things that have to change / be done in prep for the new tax season

Reply:
That is a good point. I have notes on a lot of stuff, but this is one that I also noted should be added to a yearly list lol. I will keep the list!